lib/api/controllers/securityController.js

```diff
@@ -175,4 +182,186 @@ module.exports = function SecurityController (kuzzle) {
       return q(response);
     });
   };
+
+  /**
+   * Given a user id, returns the matching User object
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getUser = function (requestObject) {
+    kuzzle.pluginsManager.trigger('security:getUser', requestObject);
+
+    if (!requestObject.data._id) {
+      return q.reject(new BadRequestError('No user id given'));
+    }
+
+    return kuzzle.repositories.user.load(requestObject.data._id)
+      .then(user => {
+        var response;
+
+        if (!user) {
+          return q(new ResponseObject(requestObject, {found: false}));
+        }
+
+        if (requestObject.data.body.hydrate !== undefined && requestObject.data.body.hydrate === false) {
```

Comment:
Example: the backoffice needs to be able to retrieve user information to display the current user and to allow an admin to edit other users.